src/features/environment.js

```diff
@@ -16,8 +16,10 @@
 import {BackSide, BoxBufferGeometry, Color, Mesh, ShaderLib, ShaderMaterial, UniformsUtils} from 'three';
 
 import {$needsRender, $onModelLoad, $renderer, $scene, $tick} from '../model-viewer-base.js';
+import {BASE_SHADOW_OPACITY} from '../three-components/StaticShadow.js'
```

Comment:
I may be biased due to my history with `features/environment`, but could the shadow feature live in a different feature? This is the first attribute that's outside of the skybox -- while it handles IBL, I wonder if a `lights` feature would be appropriate for the shadow setting, as well as other future attributes? Just spitballing (more than OK for now)

Reply:
Yeah, will move per suggestion below

Reply:
Also re: lights, the thought occurred to me as well. The main point of uncertainty for me is that technically the environment is also lighting (in graphics contexts, at least).